Updated installation.md

cameronmaguire opened the pull request:
Added a textual decomposition of expected layout of the oop-docker folder for better understandability.
Added a note about where to run the RabbitMQ commands.
Removed sudo from commands, as there's aren't required when running on windows.
Updated host to hostname.
Clarified where the database creation commands should be run.

Commit:
Updated installation.md

Added a textual decomposition of expected layout of the oop-docker folder for better understandability.
Added a note about where to run the RabbitMQ commands.
Removed sudo from commands, as there's aren't required when running on windows.
Updated host to hostname.
Clarified where the database creation commands should be run.

jackregnart commented on pages/installation.md:
```diff
@@ -133,17 +146,19 @@ Assuming that you have RabbitMQ install correctly (and added to PATH) and the `r
 ###### Import the queue config
 
-There is a pre-configured queue config in the [oop-queue-config](https://github.com/open-interop/oop-queue-config) repository. Download `rabbit-config.json` from the repo and run `sudo rabbitmqadmin import rabbit-config.json` to import it.
+There is a pre-configured queue config in the [oop-queue-config](https://github.com/open-interop/oop-queue-config) repository. Download `rabbit-config.json` from the repo and run `rabbitmqadmin import rabbit-config.json` to import it.
```
Again break this up so there is a distinction between unix-based and windows-based systems as one needs the sudo command and the other doesn't.

Commit:
Amendments to documentation changes

Re-added sudo and stated windows users should remove sudo. Removed a recommendation to run commands from sbin, as a recommendation to add rabbitmqctl is already mentioned in the documentation, plsu a few grammatical errors have been corrected.
